js/constants.js

```diff
@@ -256,3 +256,8 @@ export const MAP_COORDINATES = {
   LNG: 78.476681027237,
   LAT: 22.1991660760527,
 };
+export const BASE_URL = 'https://api.realdevsquad.com/';
+export const JOIN_POST_URL = BASE_URL + 'users/self/intro';
+export const SELF_URL = BASE_URL + 'users/self';
+export const LOGIN_URL =
+  'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97';
```

Comment:
please this from this PR I'm raising another one to fix the issue and do the changes in other files requiring this variable.

Reply:
I could not understand